sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -75,6 +89,22 @@ void queueClientWithViaQueueName() {
         assertNotNull(client);
     }
 
+    @Test
+    void topicClientWithViaTopicName() {
+        // Arrange
+        final ServiceBusSenderClientBuilder builder = new ServiceBusClientBuilder()
+            .connectionString(NAMESPACE_CONNECTION_STRING)
+            .sender()
+            .topicName(TOPIC_NAME)
+            .viaTopicName(TOPIC_NAME);
+
+        // Act
+        final ServiceBusSenderAsyncClient client = builder.buildAsyncClient();
+
+        // Assert
+        assertNotNull(client);
```

Comment:
May be we can add one more validation that "By adding 'viaTopicName` did not change the `topicName`.  You can use `client.getEntityPath()`  to validate that.

Reply:
Added assertion